src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -1464,15 +1467,17 @@ impl EventProcessor {
             value = unsafe { value.offset(1) };
         }
 
-        if mouse_delta != (0.0, 0.0) {
+        if mouse_delta.0.abs() as f32 >= NOTGULL_THRESHOLD
+            || mouse_delta.1.abs() as f32 >= NOTGULL_THRESHOLD
```

Comment:
I don't quite like this logic in-particular. Could you instead check if the given value is small enough and null it? Then just send the thing if something is not null, since those low values don't make much sense in the first place.